hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -421,3 +460,28 @@ func generateSSHSecret(data []byte, namespace string) (*corev1.Secret, error) {
 	secret.Type = corev1.SecretTypeOpaque
 	return secret, nil
 }
+
```

Comment:
may this could be simpler with"
```
func namespacedName(obj metav1.Object) types.NamespacedName {
	return types.NamespacedName{
		Namespace: obj.GetNamespace(),
		Name:      obj.GetName(),
	}
}
```

then:
```
namespacedName(o.Meta).String()
```

Reply:
cool, I like that

Reply:
updated